From 326ccf57395017d6105a04054dc634f51537f9be Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Fri, 28 Apr 2023 15:16:44 -0700
Subject: [PATCH] Loosen Python deps

pytest, python-coverage are only needed for testing. Don't break the
build if they can't be found.
---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 50bd1b5b192e..b5979a8bc4b9 100644
--- a/meson.build
+++ b/meson.build
@@ -23,8 +23,8 @@ bash_completions_dir = completions.get_variable(pkgconfig: 'completionsdir', def
 # Order: Fedora/Mageia/openSUSE || Debian/Ubuntu
 pyflakes = find_program('pyflakes-3', 'pyflakes3', required: false)
 pycodestyle = find_program('pycodestyle-3', 'pycodestyle', 'pep8', required: false)
-pytest = find_program('pytest-3', 'pytest3')  # also requires the pytest-cov plugin
-pycoverage = find_program('python3-coverage')
+pytest = find_program('pytest-3', 'pytest3', 'pytest', required: false)  # also requires the pytest-cov plugin
+pycoverage = find_program('python3-coverage', required: false)
 pandoc = find_program('pandoc', required: false)
 find = find_program('find')
 
-- 
2.40.1

